shared/src/background.js

```diff
@@ -192,7 +192,7 @@ function kagiSummarize(info, tab) {
 function kagiImageSearch(info, tab) {
   const imageUrl = info.srcUrl;
   browser.tabs.create({
-    url: `https://kagi.com/images?q=${encodeURIComponent(imageUrl)}`,
```

Comment:
I noticed that the searched URL is not encoded when using Reverse Image Search directly through the website.